packages/component-submission/server/use-cases/helpers/manuscript.js

```diff
@@ -77,18 +77,14 @@ class ManuscriptHelper {
     )
 
     try {
-      let manuscript = await ManuscriptModel.find(manuscriptId, this.userId)
+      const manuscript = await ManuscriptModel.find(manuscriptId, this.userId)
```

Comment:
we still need this for the permissions check

Reply:
Can we clarify that somehow - it's not clear that this is the step that performs the permissions check

Reply:
Ideally this should be extracted to a util function given all the use-cases will need it eventually. Also when we introduce the next stage of users and staff it would be easier to replace one function than all of these individual checks.

Reply:
Yeah, I guess I'm just not a massive fan of using throws to express logic like this, it's not clear what's going on.

Reply:
I'm happy to have a `Manuscript` function called `checkPermissions` which does this